model/messages/parsedmessage.go

```diff
@@ -0,0 +1,55 @@
+package messages
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-pg/pg/v10"
+	"go.opencensus.io/tag"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/trace"
+	"go.opentelemetry.io/otel/label"
+
+	"github.com/filecoin-project/sentinel-visor/metrics"
+)
+
+type ParsedMessage struct {
+	Cid string `pg:",pk,notnull"`
+
+	From   string `pg:",notnull"`
+	To     string `pg:",notnull"`
+	Value  string `pg:",notnull"`
+	Method string `pg:",notnull"`
+
+	Params string `pg:""`
```

Comment:
Params could be: 

```
Params []byte `pg:",type:jsonb,notnull"`
```

Making it []byte would save an allocation in the MessageProcessor

Reply:
do you mean in the task? the `string()` cast shouldn't lead to a memory copy, if you do.

Reply:
If that were true then you would be able to edit a string by modifying the byte slice's backing array.